src/Raven.Client/Documents/Commands/DeleteDocumentCommand.cs

```diff
@@ -17,6 +17,7 @@ public DeleteDocumentCommand(string id, long? etag)
 
             _id = id;
             _etag = etag;
+            ResponseType = RavenCommandResponseType.Empty;
```

Comment:
Maybe RavenCommand can set ResponseType to Empty automatically? And RavenCommand<T> to Object?